crypto/err/err.c

```diff
@@ -429,19 +433,6 @@ static unsigned long get_error_values(int inc, int top, const char **file,
     if (es == NULL)
         return 0;
 
-    if (inc && top) {
```

Comment:
removing this at the same time as adding an enumeration seems strange

Reply:
Well, it's understandable in a way... Since there is no enumeration corresponding to both flags being set, this code section is practically dead code...